keyboards/xd75/keymaps/jarred/rules.mk

```diff
@@ -0,0 +1,3 @@
+ifndef QUANTUM_DIR
```

Comment:
Could you remove this (ifndef/include/endif) block? 

It's not needed, and is a holdover from the old build system.

Reply:
File can be deleted, nothing going on here.